examples/quick_start.ipynb

```diff
@@ -0,0 +1,260 @@
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "TNkhz_nrB2lx"
+      },
+      "outputs": [],
+      "source": [
+        "# A simple update loop.\n",
+        "for _ in range(1000):\n",
+        "  grads = jax.grad(compute_loss)(params, xs, ys)\n",
+        "  updates, opt_state = optimizer.update(grads, opt_state)\n",
+        "  params = optax.apply_updates(params, updates)\n",
+        "\n",
+        "assert jnp.allclose(params, target_params), 'Optimization should result in target params'"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XXEz3j7wPZUH"
+      },
+      "source": [
+        "### Custom optimizers\n",
+        "\n",
+        "Optax makes it easy to create custom optimizers by `chain`ing gradient transforms. For example, this creates an optimizer based on `adam`. Note that the scaling is `-learning_rate` which is an important detail since `apply_updates` is additive."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "KQNI2P3YEEgP"
+      },
+      "outputs": [],
+      "source": [
+        "# Exponential decay of the learning rate.\n",
+        "scheduler = optax.exponential_decay(\n",
+        "    -start_learning_rate,  # Note that we scale updates by -learning_rate.\n",
+        "    transition_steps=1000,\n",
+        "    decay_rate=0.99)\n",
+        "\n",
+        "# Combining gradient transforms using `optax.chain`.\n",
+        "gradient_transform = optax.chain(\n",
+        "    optax.clip_by_global_norm(1.0),  # Clip by the gradient by the global norm.\n",
+        "    optax.scale_by_adam(),  # Use the updates from adam.\n",
+        "    optax.scale_by_schedule(scheduler)  # Use the learning rate from the scheduler.\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "XGUrLKxAEO3j"
+      },
+      "outputs": [],
+      "source": [
+        "# Initialize parameters of the model + optimizer.\n",
+        "params = jnp.array([0.0, 0.0])\n",
+        "opt_state = gradient_transform.init(params)\n",
+        "\n",
+        "# A simple update loop.\n",
+        "for _ in range(1000):\n",
+        "  grads = jax.grad(compute_loss)(params, xs, ys)\n",
+        "  updates, opt_state = gradient_transform.update(grads, opt_state)\n",
+        "  params = optax.apply_updates(params, updates)\n",
+        "\n",
+        "assert jnp.allclose(params, target_params), 'Optimization should result in target params'"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "pIxKL7WsXFl8"
+      },
+      "source": [
+        "## Advanced usage of `optax`"
```

Comment:
Suggestion: I think it's Optax (name) vs package name `optax` (https://github.com/deepmind/optax/blob/master/README.md)

```suggestion
        "## Advanced uses of Optax"
```